<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>623</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Моніторинг навколишнього середовища з допомогою безпровідних мереж</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>971</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="build_network_button">
       <property name="text">
        <string>Побудувати мережу</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QPushButton" name="start_modeling_button">
       <property name="text">
        <string>Розпочати моделювання</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>90</y>
      <width>981</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGraphicsView" name="area_graphics_view"/>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Параметри середовища</string>
    </property>
    <addaction name="weather_menu"/>
    <addaction name="cdo_menu"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Параметри сенсора</string>
    </property>
    <addaction name="processor_menu"/>
    <addaction name="transfer_menu"/>
    <addaction name="energy_menu"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Параметри моделювання</string>
    </property>
    <addaction name="modeling_menu"/>
    <addaction name="image_menu"/>
    <addaction name="random_menu"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Результати моделювання</string>
    </property>
    <addaction name="graph_energy_menu"/>
    <addaction name="evaluate_menu"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="weather_menu">
   <property name="text">
    <string>Метеоумови</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+W</string>
   </property>
  </action>
  <action name="cdo_menu">
   <property name="text">
    <string>Хімічно-небезпечний об'єкт</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
  <action name="processor_menu">
   <property name="text">
    <string>Процесор</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+P</string>
   </property>
  </action>
  <action name="transfer_menu">
   <property name="text">
    <string>Характеристики передачі</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+T</string>
   </property>
  </action>
  <action name="energy_menu">
   <property name="text">
    <string>Енергетичні величини</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+E</string>
   </property>
  </action>
  <action name="modeling_menu">
   <property name="text">
    <string>Параметри мережі</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+M</string>
   </property>
  </action>
  <action name="image_menu">
   <property name="text">
    <string>Відображення</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+I</string>
   </property>
  </action>
  <action name="graph_energy_menu">
   <property name="text">
    <string>Графік споживання енергії</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+G</string>
   </property>
  </action>
  <action name="evaluate_menu">
   <property name="text">
    <string>Оцінка хімічного становища</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+O</string>
   </property>
  </action>
  <action name="random_menu">
   <property name="text">
    <string>Генератор випадкових чисел</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
